.github/workflows/CI.yml

```diff
@@ -8,82 +8,93 @@ on:
     branches: [ main, develop ]
```

Comment:
This is triggering on PRs and pushed to `main` and `develop`.
Do we really need them for the `develop` branch? 
Is the `develop` branch currently used as a "staging" branch? 
@blimlim @truth-quark

Reply:
I think the `develop` branch has been deprecated and PRs are now done directly to `main`, so it should be save to remove the `develop` branch here.

Reply:
Removed `develop` in [cdac87bff05ba6f6e83d99a462bb1d9f81003406](https://github.com/ACCESS-NRI/um2nc-standalone/pull/146/commits/cdac87bff05ba6f6e83d99a462bb1d9f81003406)